deployment/terraform/create.go

```diff
@@ -220,6 +227,37 @@ func (t *Terraform) Create(initData bool) error {
 		if err := pingServer("http://" + pingURL); err != nil {
 			return fmt.Errorf("error whiling pinging server: %w", err)
 		}
+
+		if t.output.HasKeycloak() {
+			keycloakCmdFmt := "/opt/mattermost/bin/mmctl config set %s '%s' --local"
```

Comment:
All of this is actually done by setting the model.Config struct. See `(t *Terraform) updateAppConfig`. This code should exist inside that method.